src/OpenLoco/GameCommands/LoadSaveQuit.cpp

```diff
@@ -0,0 +1,323 @@
+    {
+        registers regs;
+        call(0x00441843, regs);
+        return regs.eax;
+    }
+
+    // 0x004417A7
+    static bool loadLandscapeOpen()
+    {
+        registers regs;
+        call(0x004417A7, regs);
+        return regs.eax;
+    }
+
+    // 0x00441993
+    static bool saveLandscapeOpen()
+    {
+        registers regs;
+        call(0x00441993, regs);
+        return regs.eax;
+    }
+
+    // 0x00441FA7
+    static bool sub_441FA7()
+    {
+        registers regs;
+        call(0x00441FA7);
+        return regs.eax != 0;
+    }
+
+    // 0x004424CE
+    static bool sub_4424CE()
+    {
+        registers regs;
+        call(0x004424CE);
+        return regs.eax != 0;
+    }
+
+    // 0x0043BFF8
+    static uint32_t loadGame()
+    {
+        GameCommands::do_21(1, 0);
+        Input::toolCancel();
+
+        if (isEditorMode() && loadLandscapeOpen())
+        {
+            // 0x0043C087
+            auto path = fs::path(&_savePath[0]).replace_extension(S5::extensionSC5).u8string();
+            std::strncpy(&_currentScenarioFilename[0], path.c_str(), std::size(_currentScenarioFilename));
+
+            if (sub_4424CE())
+            {
+                resetScreenAge();
+                throw GameException::Interrupt;
+            }
+        }
+        else if (!isNetworked() && loadSaveGameOpen())
+        {
+            // 0x0043C033
+            auto path = fs::path(&_savePath[0]).replace_extension(S5::extensionSV5).u8string();
+            std::strncpy(&_currentScenarioFilename[0], path.c_str(), std::size(_currentScenarioFilename));
+
+            if (sub_441FA7())
+            {
+                resetScreenAge();
+                throw GameException::Interrupt;
+            }
+        }
+        else if (isNetworked())
+        {
+            // 0x0043C0DB
+            if (CompanyManager::getControllingId() == CompanyManager::updatingCompanyId())
+            {
+                MultiPlayer::setFlag(MultiPlayer::flags::flag_4);
+                MultiPlayer::setFlag(MultiPlayer::flags::flag_3);
+            }
+        }
+
+        // 0x0043C0D1
+        Gfx::invalidateScreen();
+        return 0;
+    }
+
+    // 0x0043C182
+    static uint32_t quitGame()
+    {
+        _game_command_nest_level = 0;
+
+        // Path for networked games; untested.
+        if (isNetworked())
+        {
+            clearScreenFlag(ScreenFlags::networked);
+            auto playerCompanyId = CompanyManager::getControllingId();
+            auto previousUpdatingId = CompanyManager::updatingCompanyId();
+            CompanyManager::updatingCompanyId(playerCompanyId);
+
+            Ui::WindowManager::closeAllFloatingWindows();
+
+            CompanyManager::updatingCompanyId(previousUpdatingId);
+            setScreenFlag(ScreenFlags::networked);
+
+            // If the other party is leaving the game, go back to the title screen.
+            if (playerCompanyId != previousUpdatingId)
+            {
+                // 0x0043C1CD
```

Comment:
Do we know what the missing function call to `0x406B2F` was?

Reply:
No, to be honest. It seemed like a wrapper for Win32-related cleanup logic. I haven't spent much on it, other than that.